stable/chartmuseum/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
+name: chartmuseum
+version: 0.5.0
```

Comment:
should this be a major version bump (1.0.0?). I think a minor version bump is okay for `0.` releases, but perhaps the major version would be more indicative of some of these proposed value changes.

Reply:
I was wondering something similar. If we are changing helm values YAML keys, then there's backwards incompatibility at those points, so that requires a semver MAJOR bump.